class.js

```diff
@@ -0,0 +1,63 @@
+let currendID = 0;
+
+let listBook = null;
+
+function structure(title, author) {
+  const itemId = currendID;
+
+  const listBooks = document.querySelector('.list-book');
+  const li = document.createElement('li');
+  li.id = currendID;
+  const pTitleAuthor = document.createElement('p');
+  pTitleAuthor.textContent = `"${title}" by ${author}`;
+
+  const btnRemove = document.createElement('button');
+  btnRemove.textContent = 'Remove';
+
+  btnRemove.addEventListener('click', () => {
+    listBook.delete(itemId);
+  });
+
+  li.append(pTitleAuthor, btnRemove);
+
+  listBooks.append(li);
+  currendID += 1;
+}
+
+class ListBooks {
+  constructor() {
+    this.bookcollection = [];
+  }
+
+  add(title, author) {
+    const book = { id: currendID, title, author };
+    this.bookcollection.push(book);
+    structure(title, author);
+  }
+
+  delete(id) {
+    document.getElementById(id).remove();
+    this.bookcollection = this.bookcollection.filter((book) => book.id !== id);
+  }
+}
+listBook = new ListBooks();
+
+const button = document.querySelector('#button');
+
+button.addEventListener('click', () => {
+  const title = document.getElementById('Title').value;
+  const author = document.getElementById('Author').value;
+  listBook.add(title, author);
+});
+
```

Comment:
[OPTIONAL]
- [ ] It would be great to clear the inputs once a book is added to the list.
`hint: You may do something like:`
`After the event is fired, I mean after adding the book in the array of books, set the title = ' ' and the author as well`

- [ ] Also for a better user experience, you may consider using the `keypress` event as well to add the book so that when the user hits `Enter` the book may be added as well.
`Hint: You may consider grabbing the user inputs in a function and be calling that function whenever the Add button is clicked or whenever Enter is pressed` :smile:

Reply:
The changes have been done. Thanks you for the suggestions 👍